kubernetes_client.go

```diff
@@ -341,3 +363,94 @@ func (c *kubernetesClient) DrainKubeDNSFromNode(ctx context.Context, nodeName st
 
 	return
 }
+
+func (c *kubernetesClient) evictPods(ctx context.Context, pods []v1.Pod, stop <-chan bool) (lastErr error) {
+	podsPerBatch := 10
```

Comment:
Why 10 in particular? Should this be configurable?

Reply:
I pulled it out of thin-air as what I thought might be a reasonable number of eviction requests to make at a time. We could configure it, not sure if it would be valuable or not. What do you think?

Reply:
I'm not sure it would be either, it could probably be more but we can let the upstream folks look at that if they want to. This seems reasonable, more just curious if there was a limit hit here or anything